src/map_main.cpp

```diff
@@ -76,7 +77,7 @@ int pandora_map(int argc, char *argv[]) {
     // otherwise, parse the parameters from the command line
     string prgfile, reads_filepath, outdir = "pandora", vcf_refs_file, log_level="info";
     uint32_t w = 14, k = 15, min_cluster_size = 10, genome_size = 5000000, max_covg = 300,
-            min_allele_covg_gt = 0, min_total_covg_gt = 0, min_diff_covg_gt = 0, min_kmer_covg=0; // default parameters
+            min_allele_covg_gt = 0, min_total_covg_gt = 0, min_diff_covg_gt = 0, min_kmer_covg=0, threads=1; // default parameters
```

Comment:
threads can probably be a `unint8_t` right? I mean I doubt anyone is going to ask for >255 threads...